doc/src/Intro.rst

```diff
@@ -44,6 +52,10 @@ The BCM modules provide some high-level cmake functions to take care of all the
 
     bcm_auto_pkgconfig()
 
+.. The above snippet is mysterious for folks not accustomed to this library.
+   Where are the '.cpp' files being added? Where are the include files? What
+   do these functions do?
```

Comment:
Yes, I need to explain this better. This relies on boost's directory structure to find the includes. Of course, Boost.Core does not have any source files, so that is why it is not mentioned here.